showcase/scripts/verify.sh

```diff
@@ -12,16 +12,26 @@ readonly REPLACE_SOURCE='false'
 readonly PERFORM_CLEANUP='false'
 bash "${SCRIPT_DIR}/generate_components.sh" "${REPLACE_SOURCE}" "${PERFORM_CLEANUP}"
 
-# compare with proto library
 PROTO_PROJECT_DIR='proto-gapic-showcase-v1beta1'
-diff -ru "${SHOWCASE_DIR}/${PROTO_PROJECT_DIR}/src" "${SCRIPT_DIR}/showcase-output/proto-showcase-output/src"
-
-# compare with grpc library
 GRPC_PROJECT_DIR='grpc-gapic-showcase-v1beta1'
-diff -ru "${SHOWCASE_DIR}/${GRPC_PROJECT_DIR}/src/main/java/com" "${SCRIPT_DIR}/showcase-output/grpc-showcase-output/src/main/java/com"
-
-# compare with gapic library
 GAPIC_PROJECT_DIR='gapic-showcase'
-diff -ru "${SHOWCASE_DIR}/${GAPIC_PROJECT_DIR}/src/main/java" "${SCRIPT_DIR}/showcase-output/gapic-showcase-output/src/main/java"
+
+{
+  # compare with proto library
+  (diff -ru "${SHOWCASE_DIR}/${PROTO_PROJECT_DIR}/src" "${SCRIPT_DIR}/output/showcase-output/proto-showcase-output/src") &&
+
+  # compare with grpc library
+  (diff -ru "${SHOWCASE_DIR}/${GRPC_PROJECT_DIR}/src/main/java/com" "${SCRIPT_DIR}/output/showcase-output/grpc-showcase-output/src/main/java/com") &&
+
+  # compare with gapic library
+  (diff -ru "${SHOWCASE_DIR}/${GAPIC_PROJECT_DIR}/src/main/java" "${SCRIPT_DIR}/output/showcase-output/gapic-showcase-output/src/main/java") 
+} || {
+  failure="true"
```

Comment:
Is this logic correct? I see that showcase integration tests always exit with 1.

Reply:
It is correct, I found out why they were failing. It was because there was a `pushd cd` (fixed in https://github.com/googleapis/sdk-platform-java/pull/1987/commits/0b54240a419e089552a7491147c195e3366eaa23)

Now `mvn verify -P enable-golden-tests` behaves correctly